docs/store/datastore.md

```diff
@@ -259,6 +259,16 @@ import os
 os.environ["HADOOP_USER_NAME"] = "..."
 ```
 
+To register an HDFS datastore profile and use it as described in [Data store profiles](#data-store-profiles), use:
```

Comment:
It's a docs-wide and even code-wide issue, but it seems we can't decide if it's a datastore or a data store. Here it stands out because both are used in the same sentence.

Reply:
in the docs it should be "data store"
Updated accordingly